modules/nodes-pods-autoscaling-custom-trigger.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-custom.adoc
+
+:_content-type: CONCEPT
+[id="nodes-pods-autoscaling-custom-trigger_{context}"]
+= Understanding the custom metrics autoscaler triggers
+
+Triggers, also known as scalers, provide the metrics that the Custom Metrics Autoscaler Operator uses to scale your pods. You can specify a single trigger or multiple triggers. When using multiple triggers, the scaling is based on the greatest value from all the triggers.
+
+This section contains examples of some triggers you can use with the custom metrics autoscaler (CMA). For a full list of available triggers, see the link:https://keda.sh/docs/2.7/scalers[KEDA Scalers] documentation. Because the custom metrics autoscaler is Technology Preview, Red Hat does not recommend using the triggers/scalers in production. 
```

Comment:
Is Red Hat supporting all of the Scalers listed by the upstream? IE: if a customer finds a bug with one of these scalers (or the upstream adds a scaler in the middle of a 4.Y version's lifecycle) can a customer take advantage of it? Can they get patches and security updates for the scalers?

If the answer is no; to this then we shouldn't like to the upstream here; but instead we should list what Red Hat is supporting.

All in all I don't have an issue linking to the upstream here; but we have to clarify this paragraph with a support clause / call out that said (more or less): Red Hat provides and only supports the ability to use scalers; the source / support for each individual scaler is offered by the provider of at scaler (please see the provider for support options), etc, etc.